src/components/FilterBar.jsx

```diff
@@ -1,11 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { Grid, FormControl, Select, MenuItem } from '@mui/material';
+import {
+  Grid,
+  FormControl,
+  Select,
+  MenuItem,
+  Button,
+  Popover,
+} from '@mui/material';
 import AmountFilter from './AmountFilter';
 import GradeLevelFilter from './GradeLevelFilter';
 import qParams from '../lib/QueryParams';
 import sortOptions, { DEADLINE_ASC } from '../lib/sortOptions';
 import MajorFilter from './MajorFilter';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+
+const Filter = ({ title, filter }) => {
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  return (
+    <>
+      <Button
+        variant="outlined"
+        onClick={(e) => setAnchorEl(e.currentTarget)}
+        endIcon={<ArrowDropDownIcon color="primary" />}
+        sx={{ m: 1, height: (theme) => theme.spacing(4) }}>
+        {title}
+      </Button>
+      <Popover
+        open={Boolean(anchorEl)}
+        anchorEl={anchorEl}
+        onClose={() => setAnchorEl(null)}
+        anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
+        transformOrigin={{ vertical: 'top', horizontal: 'left' }}
+        PaperProps={{ style: { minWidth: 280 } }}>
```

Comment:
Hmm, for the major filters this makes it take up an arbitrary width. I suggest either:
- Setting a `maxWidth` OR
- Stacking major selections on top of each other

Reply:
The reason I made it this width was so that you can seel the Input's placeholder.

Reply:
<img width="248" alt="Screen Shot 2022-01-25 at 9 29 33 AM" src="https://user-images.githubusercontent.com/40483569/151028166-25047006-2018-46cd-8f00-626f76f72c51.png">

VS

<img width="281" alt="Screen Shot 2022-01-25 at 9 30 07 AM" src="https://user-images.githubusercontent.com/40483569/151028257-f7bf48e9-7041-42f1-8ed8-935c83549b7e.png">

Reply:
Sorry I meant the width is arbitrary when multiple majors are selected:
<img width="1407" alt="Screen Shot 2022-01-25 at 5 37 06 PM" src="https://user-images.githubusercontent.com/8023233/151088684-bc049e22-9c03-43e6-b9cd-efcd18415e06.png">

I think a quick fix would be to additionally specify `maxWidth` to something reasonable that fits the needed widths for the other filters. 

Or actually, since only the major filter needs this, you could add an optional `width` (or `popoverWidth` since it's the width of the popover) property to the `FilterButton` component that defaults `undefined`.